bin/install-dependencies.bash

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+# Installs dependency for Ubuntu Linux, which is used on Github Actions
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Print functions
+stdmsg() {
+  local IFS=' '
+  printf '%s\n' "$*"
+}
+
+errmsg() {
+  stdmsg "$*" 1>&2
+}
+
+# Trap exit handler
+trap_exit() {
+  # It is critical that the first line capture the exit code. Nothing
+  # else can come before this. The exit code recorded here comes from
+  # the command that caused the script to exit.
+  local exit_status="$?"
+
+  if [[ ${exit_status} -ne 0 ]]; then
+    errmsg 'There is an error installing the dependencies.'
+    exit 1
+  fi
+}
+trap trap_exit EXIT
+
+# Update package list and install curl and shellcheck
+sudo apt-get update
+sudo apt-get install curl shellcheck -y
```

Comment:
By the way, I noticed that [the 24.04 runner image comes with curl and shellcheck preinstalled](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md).

It won't hurt performance to leave them in this `apt-get` command. `apt-get` doesn't reinstall already-installed packages unless you pass the `--reinstall` flag.